tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -474,3 +474,21 @@ Given('the user has created the following files', function (entryList) {
   })
   return client
 })
+
+When('the user browses to the folder {string} on the files page', (folderName) => {
+  const targetFolder = folderName === '/' ? '' : folderName
+  return client
+    .page.filesPage()
+    .navigateAndWaitTillLoaded(targetFolder)
+})
+When('the user copies the permalink of the current folder using the webUI', function () {
+  return client.page.filesPage().copyPermalinkFromFilesAppBar()
+})
+Then('the clipboard content should match current folder permalink', function () {
+  return client.getClipBoardContent(function (value) {
+    webdav.getProperties('/', client.globals.currentUser, ['oc:privatelink'])
```

Comment:
this gets the permalink of "/" but we need it of the "current folder", I expect that to fail for the subfolder test
What about passing in the resource name is the function?

```
Then('the clipboard content should match the permalink of resource {string}', function (resource) {
```

Reply:
done